docs/plugins.md

```diff
@@ -0,0 +1,51 @@
+# Feather - Quill - Plugins
+Feather has support for plugins. They can either be run as native code, or through WebAssembly using [Wasmer](https://github.com/wasmerio/wasmer). The API for the two options are the same, but wasmer has a few benefits for the cost of slightly worse performance. ([More details](#Wasm-or-Native)).
+
+## Getting Started
+
+# Building and Running
+To build a quill plugin you will need a tool called [cargo-quill](https://github.com/¨-rs/feather/tree/main/quill/cargo-quill). To install cargo quill you need the [rust build tools](https://www.rust-lang.org/tools/install), and clone Feathers [git repository](https://github.com/feather-rs/feather).
+
+Once that is setup, locate the folder `feather/quill/cargo_quill` in a terminal. Then run the following command.
+
+```bash
+cargo install --path <path to quill>
```

Comment:
What is "<path to quill>"? Do I need to build cargo-quill first?

Reply:
If i change it to cargo-quill, does that  spark joy?

Reply:
I'm just not sure why you're using a placeholder. You could say "In the root directory of feather, run the following command" followed by `cargo install --path quill/cargo_quill/` or you could also say "Navigate into quill/cargo_quill and run the following command", followed by `cargo install --path`.